packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/services-error-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Button, Modal } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { getTitle } from './constants';
+import AuthorizationRequiredModal from './authorization-required-modal.jsx';
+import InsufficientBalanceModal from './insufficient-balance-modal.jsx';
+import CompanyWideLimitExceededModal from './company-wide-limit-exceeded-modal.jsx';
+
+type TServicesError = {
+    code: string;
+    message: string;
+    type?: string;
+};
+
+type TPropServicesErrorModel = {
+    is_virtual?: boolean;
+    is_visible: boolean;
+    is_logged_in?: boolean;
+    onConfirm: () => void;
+    services_error: TServicesError;
+};
+
+const ServicesErrorModal = ({
+    is_virtual,
+    is_visible,
+    is_logged_in,
+    onConfirm,
+    services_error,
+}: TPropServicesErrorModel) => {
+    const { code, message, type } = services_error;
+
+    if (!code || !message) return null;
```

Comment:
```suggestion
    if (!code || !message) return <React.Fragment />;
```

use the React Fragment so the child props are rendered properly.
Becasuse the by returning `null` you are disconnecting the updating process of `DOM`